ifrs17/Import/Importers.ipynb

```diff
@@ -978,35 +1011,31 @@
             "source": [
                 "Import.DefineFormat(ImportFormats.Cashflow, async (options, dataSet) => {",
                 "\n    Activity.Start();",
-                "\n    var args = (await GetArgsFromMainAsync<PartitionByReportingNodeAndPeriod>(dataSet)) with {ImportFormat = ImportFormats.Cashflow};",
-                "\n    await DataNodeFactoryAsync(dataSet, ImportFormats.Cashflow, args);",
+                "\n    var mainArgs = (await GetArgsFromMainAsync<PartitionByReportingNodeAndPeriod>(dataSet)) with {ImportFormat = ImportFormats.Cashflow};",
+                "\n    if(Activity.HasErrors()) return Activity.Finish();",
+                "\n    await DataNodeFactoryAsync(dataSet, ImportFormats.Cashflow, mainArgs);",
                 "\n    if(Activity.HasErrors()) return Activity.Finish();",
                 "\n    ",
                 "\n    var workspace = Workspace.CreateNew();",
-                "\n    var parsingLog = await ParseCashflowsToWorkspaceAsync(dataSet, args, workspace);",
-                "\n    if(parsingLog.Errors.Any()) return Activity.Finish().Merge(parsingLog);",
+                "\n    var log = await ParseCashflowsToWorkspaceAsync(dataSet, mainArgs, workspace);",
+                "\n    if(log.Errors.Any()) return Activity.Finish().Merge(log);",
+                "\n",
+                "\n    var targetArgs = mainArgs.Scenario == null ",
+                "\n                     ? mainArgs.RepeatOnce().Concat(await DataSource.Query<PartitionByReportingNodeAndPeriod>()  // TODO: check that concat is adding at the end of the list",
```

Comment:
remove TODO since we have already tested this

Reply:
done